src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -267,6 +269,10 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
                 tabControl.setTab(tabControl.TABS.DIRECTIONS);
             }
         }
+
+        if (key === 'origin') {
```

Comment:
Put it here because the places need to be updated for either tab; in case user changes origin in directions tab, then switches back to explore.